src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html

```diff
@@ -51,6 +51,22 @@
     </ul>
 </app-page-options-section>
 
+<app-page-options-section *ngIf="associatedPullRequests.length > 0" sectionName="Associated Pull Requests">
+    <ul class="list-group">
+        <li class="list-group-item" *ngFor="let pullRequest of associatedPullRequests">
+            <a href="https://github.com/{{pullRequest.repoName}}/pull/{{pullRequest.pullRequestNumber}}" target="_blank">{{pullRequest.repoName}}/{{pullRequest.pullRequestNumber}}</a>
+        </li>
+    </ul>
+</app-page-options-section>
+
+<app-page-options-section *ngIf="pullRequestsOfAssociatedAPIRevisions.length > 0" sectionName="Associated APIRevisions" [collapsedInput]="true">
```

Comment:
`Associated APIRevisions` is the section name.  APIRevisions is a DB name and I don't think that's the label we should use on UI if label and section name are same. We can just say `Associated Revisions`

Reply:
I think APIRevisoin is fine to differentiate from Sample revision, by the way that's the current name in the old UI.

Reply:
`Associated API Revisions`